### ISSUE TITLE
packetevents warning spam with viaversion 4.10.1

### ISSUE THREAD

**Issue by Jcodeerd** [mcpvp-club/FasterCrystals#13]
Expected behaviour: No errors when using viaversion

Observed behaviour:
```
[01:37:48 INFO]: [FasterCrystals] [STDOUT] Unable to grab ViaVersion client version for player!
[01:37:48 INFO]: [packetevents] Unable to grab ViaVersion client version for player!
[01:37:48 INFO]: [packetevents] Unable to grab ViaVersion client version for player!
[01:37:48 INFO]: [FasterCrystals] [STDOUT] Unable to grab ViaVersion client version for player!
[01:37:48 INFO]: [packetevents] Unable to grab ViaVersion client version for player!
[01:37:49 INFO]: [packetevents] Unable to grab ViaVersion client version for player!
[01:37:49 INFO]: [FasterCrystals] [STDOUT] Unable to grab ViaVersion client version for player!
[01:37:49 INFO]: [packetevents] Unable to grab ViaVersion client version for player!
[01:37:49 INFO]: [packetevents] Unable to grab ViaVersion client version for player!
[01:37:49 INFO]: [FasterCrystals] [STDOUT] Unable to grab ViaVersion client version for player!
[01:37:49 INFO]: [packetevents] Unable to grab ViaVersion client version for player!
```

Doesn't seem to be causing issues ingame, but it's annoying for admins since it spams the console and creates unnecessary big log files.

### Plugin versions:
Viaversion: 4.10.1
Fastercrystals: 1.4.3

**Comment by Jyguy** [comment]
Packetevents has this fixed but their snapshot builds aren't used right now. Once it updates (along with 1.20.5 update) it will be fixed.

In the meantime the only solution is downgrading viaversion.

**Comment by Jyguy** [comment]
Fixed by #14
